src/yaml/aslYamlLanguageService.ts

```diff
@@ -0,0 +1,228 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+import * as prettier from 'prettier'
+import {
+    DocumentLanguageSettings,
+    getLanguageService as getLanguageServiceVscode,
+    JSONDocument,
+    JSONSchema,
+    LanguageService,
+    LanguageServiceParams,
+} from 'vscode-json-languageservice'
+import { TextDocument } from 'vscode-languageserver-textdocument'
+import {
+    CompletionList,
+    DocumentSymbol,
+    FormattingOptions,
+    Hover,
+    Position,
+    Range,
+    SymbolInformation,
+    TextEdit
+} from 'vscode-languageserver-types'
+import {
+    parse as parseYAML,
+    SingleYAMLDocument,
+    YAMLDocument,
+} from 'yaml-language-server/out/server/src/languageservice/parser/yamlParser07'
```

Comment:
Is there a reason the naming scheme is like this? Seems a little odd to have the 07.

Reply:
Unfortunately, the YAML Language server does not export these at the service hence they are being imported this way.